hummingbot/connector/exchange/okex/okex_exchange.pyx

```diff
@@ -1026,13 +992,20 @@ cdef class OkexExchange(ExchangeBase):
     def cancel(self, trading_pair: str, client_order_id: str):
         return self.c_cancel(trading_pair, client_order_id)
 
-    def get_fee(self,
-                base_currency: str,
-                quote_currency: str,
-                order_type: OrderType,
-                order_side: TradeType,
-                amount: Decimal,
-                price: Decimal = s_decimal_NaN) -> TradeFee:
+    async def get_fee(self,
+                      base_currency: str,
+                      quote_currency: str,
+                      order_type: OrderType,
+                      order_side: TradeType,
+                      amount: Decimal,
+                      price: Decimal = s_decimal_NaN) -> TradeFee:
+        msg = await self._api_request("GET",
```

Comment:
Not a good idea to make API call to get fee data here. This get_fee is called many times and often in a loop, those caller functions will then take much longer time to complete.
It is ok to use the original implementation where the fee is estimated.

Reply:
Latest commit pushed back to original get_fee estimation method